From a1cd8a787a18da69ac4e48cfc876623bd7323799 Mon Sep 17 00:00:00 2001
From: Alan Tull <alan.tull@freescale.com>
Date: Fri, 3 Jun 2011 11:51:16 -0500
Subject: [PATCH 998/998] ENGR00144497 Revert MX5 clock: Fix ssi_ext1_clk is always on without alsa

This reverts commit 571d23cf91e5906ae61eae1201de816ed30f582e:
 "ENGR00144088 MX5 clock: Fix ssi_ext1_clk is always on without
 alsa playing"

That commit prevented the SGTL5000 driver from being able to id the part.

Signed-off-by: Alan Tull <alan.tull@freescale.com>
---
 arch/arm/mach-mx5/clock.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx5/clock.c b/arch/arm/mach-mx5/clock.c
index 1a6d5aa..c051fd8 100644
--- a/arch/arm/mach-mx5/clock.c
+++ b/arch/arm/mach-mx5/clock.c
@@ -4988,6 +4988,7 @@ int __init mx53_clocks_init(unsigned long ckil, unsigned long osc, unsigned long
 	clk_disable(&ssi_ext1_clk);
 	clk_set_parent(&ssi_ext1_clk, &pll2_sw_clk);
 	clk_set_rate(&ssi_ext1_clk, 24000000);
+	clk_enable(&ssi_ext1_clk);
 	clk_set_parent(&ssi_ext2_clk, &ssi2_clk[0]);
 
 	/* move usb_phy_clk to 24MHz */
-- 
1.7.4.1

